test/integration/testsuite_test.go

```diff
@@ -54,13 +53,9 @@ var _ = BeforeSuite(func() {
 
 	Expect(err).NotTo(HaveOccurred())
 
-	// bundle location
-	bundlePath = "embedded"
-	if !versionInfo.Embedded {
-		bundlePath = os.Getenv("BUNDLE_PATH") // this env var should contain location of bundle or string "embedded"
-		if bundlePath != "embedded" {         // if real bundle
-			Expect(bundlePath).To(BeAnExistingFile())
-		}
+	bundlePath = os.Getenv("BUNDLE_PATH") // this env var should contain location of bundle or string "embedded"
+	if bundlePath != "" {
```

Comment:
Which bundle is going to be automatically downloaded if `bundlepath == ""` is true?

Reply:
@jsliacan the one which is in the https://github.com/code-ready/crc/blob/main/pkg/crc/machine/bundle/constants.go here.